### PR TITLE
PyZQM recipe needs setuptools, list it explicitly in deps

### PR DESCRIPTION
**Pull request by rambo**
It seems to be no longer automagically available at this build stage (maybe some order of operations got changed)

### CODE REVIEW

File: pythonforandroid/recipes/pyzmq/__init__.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ class PyZMQRecipe(CythonRecipe):
     version = 'master'
     url = 'https://github.com/zeromq/pyzmq/archive/{version}.zip'
```
Comment (AndreMiras) [inline pythonforandroid/recipes/pyzmq/__init__.py:14]: Could we take the opportunity to pin this version to last release so our builds are more reproducible?
```suggestion
    version = '20.0.0'
    https://github.com/zeromq/pyzmq/archive/v{version}.zip
```

File: pythonforandroid/recipes/pyzmq/__init__.py [hunk 2]
```diff
@@ -13,7 +13,7 @@ class PyZMQRecipe(CythonRecipe):
...
     site_packages_name = 'zmq'
-    depends = ['libzmq']
+    depends = ['setuptools', 'libzmq']
```
Comment (AndreMiras) [inline pythonforandroid/recipes/pyzmq/__init__.py:16]: Probably became a hard dependency in recently released pyzmq v20.0.0

**Review by AndreMiras — DISMISSED** [review]
Thanks for the update, I've added a small suggestion if you don't mind editing the PR?

File: pythonforandroid/recipes/pyzmq/__init__.py (hunk 1 above)
Comment (rambo) [reply pythonforandroid/recipes/pyzmq/__init__.py]: Changed

**Review by AndreMiras — APPROVED** [review]
Thanks

**Pull request merged**